buildlib/az-helpers.sh

```diff
@@ -247,3 +247,23 @@ get_arch() {
         echo "$arch"
     fi
 }
+
+git_clone_with_retry() {
```

Comment:
fix indentation (see other functions in this file)

Reply:
Changed indentation to match the existing code style.

Reply:
existing indentation uses 4 spaces you use tabs, its not the same

Reply:
Good catch! Fixed.